runtime/doc/lsp.txt

```diff
@@ -2131,6 +2131,43 @@ stop({bufnr}, {client_id})                    *vim.lsp.semantic_tokens.stop()*
       • {client_id}  (`integer`) The ID of the |vim.lsp.Client|
 
 
+==============================================================================
+Lua module: vim.lsp.document_color                        *lsp-document_color*
+
+enable({enable}, {filter}, {opts})           *vim.lsp.document_color.enable()*
+    Enables document highlighting from the given language client in the given
+    buffer.
```

Comment:
In general for these kinds of features which are gated on support for a particular LSP capability (in this case, `textDocument/documentColor`) I think it'd be nice to include an example snippet that shows how to enable this feature conditionally. Or if not a full snippet, at least mentioning here in the docs which capability this feature maps to.

Maybe something like:

```suggestion
    Enables document highlighting from the given language client in the given
    buffer. Requires the `textDocument/documentColor` capability.
```